app/lib/credentialDisplay/openBadgeCredential.tsx

```diff
@@ -35,24 +36,38 @@ export const openBadgeCredentialDisplayConfig: CredentialDisplayConfig = {
 function OpenBadgeCredentialCard({ rawCredentialRecord, onPressIssuer }: CredentialCardProps): React.ReactElement {
   const { styles, theme } = useDynamicStyles(dynamicStyleSheet);
   const { credential } = rawCredentialRecord;
-  const { credentialSubject, issuer, name } = credential;
+  const { credentialSubject, issuer } = credential;
 
   const issuanceDate = getIssuanceDate(credential);
   const expirationDate = getExpirationDate(credential);
   const formattedIssuanceDate = issuanceDate ? moment(issuanceDate).format(DATE_FORMAT) : 'N/A';
   const formattedExpirationDate = expirationDate ? moment(expirationDate).format(DATE_FORMAT) : 'N/A';
 
   const {
+    issuedTo,
     description,
     criteria,
-    subjectName,
     numberOfCredits,
     startDateFmt,
     endDateFmt,
     achievementImage,
     achievementType
   } = credentialSubjectRenderInfoFrom(credentialSubject);
 
+  const isOpenBadgeCredential = credential.type.includes('OpenBadgeCredential');
+
+  const issuedto = issuedTo?.trim() 
+    ? issuedTo 
+    : isOpenBadgeCredential 
+      ? credential.name ?? null 
+      : null;
+  
+  const subjectName = isOpenBadgeCredential 
```

Comment:
There seems to be some confusion here. This is not the name of the subject, but name of the VC itself. So, please change `subjectName` to `credentialName`.